docs/core/tools/project-json-to-csproj.md

```diff
@@ -0,0 +1,592 @@
+---
+title: project.json and csproj comparison - .NET Core | Microsoft Docs
+description: See a mapping between project.json and csproj elements.
+keywords: project.json, csproj, .NET Core, MSBuild
+author: natemcmaster
+ms.author: mairaw
+ms.date: 02/17/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 79c50621-a24a-4e64-bbb9-b953113e841c
+---
+
+# A mapping between project.json and csproj properties
+
+By [Nate McMaster](http://github.com/natemcmaster)
+
+During the development of the .NET Core tooling, an important design change was made to 
+no longer support *project.json* files and instead move the .NET Core projects to the MSBuild/csproj 
+format.
+
+This article shows how the settings in *project.json* are represented in the MSBuild/csproj format so you
+can learn how to use the new format and understand the changes made by the migration tools when you're
+upgrading your project to the latest version of the tooling. 
+ 
+## The csproj format
+
+The new format, \*.csproj, is an XML-based format. The following example shows the root node of a 
+.NET Core project using the new `Microsoft.NET.Sdk` metapackage.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It may be worth noting that Web projects should use `Microsoft.NET.Sdk.Web` here.